FAQ.md

```diff
@@ -192,3 +197,97 @@ Out of convenience, one might create a symlink to a directory within their `GOPA
 
 This is the only symbolic link support that `dep` really intends to provide. In keeping with the general practices of the `go` tool, `dep` tends to either ignore symlinks (when walking) or copy the symlink itself, depending on the filesystem operation being performed.
 
+## How do I roll releases that `dep` will be able to use?
+
+In short: make sure you've committed your `Gopkg.toml` and `Gopkg.lock`, then
+just create a tag in your version control system and push it to the canonical
+location. `dep` is designed to work automatically with this sort of metadata
+from `git`, `bzr`, and `hg`.
+
+It's strongly preferred that you use [semver](http://semver.org)-compliant tag
+names. We hope to develop documentation soon that describes this more precisely,
+but in the meantime, the [npm](https://docs.npmjs.com/misc/semver) docs match
+our patterns pretty well.
+
+## How does `dep` decide what version of a dependency to use?
+
+The full algorithm is complex, but the most important thing to understand is
```

Comment:
Work on which - the algorithm? Or a corresponding FAQ entry?

If you mean the latter, https://github.com/sdboyer/gps/wiki/gps-for-Contributors#the-solving-algorithm is a rough draft that I wrote a while back explaining the rough mechanics of the algorithm. It might be useful, but frankly, if our goal is making solver mechanics comprehensible, then I suspect we'll have a hard time unless we make an actual visualization. The process is simply too complex.

The former is a _much_ larger discussion. As [Russ laid out](http://research.swtch.com/version-sat), the essential problem here is NP-complete, requiring some kind of solver. Those are complex beasts. I know I've been promising a writeup for a while of why I think it's important that we retain a solver and embrace those costs. The deadline I've set for myself with that is now the week before Gophercon.